### ISSUE TITLE
Issue deserializing LimitedUser

### ISSUE THREAD

**Issue by Rexios80** [vrchatapi/vrchatapi-dart#11]
Pasting this here so we don't forget

https://discord.com/channels/418093857394262020/873664574551912498/1192911528626749560

```
DioException [unknown]: null
Error: CheckedFromJsonException
Could not create LimitedUser.
There is a problem with "profilePicOverride".
Required keys are missing: profilePicOverride, userIcon.
#0      FriendsApi.getFriends (package:vrchat_dart_generated/src/api/friends_api.dart:339)
<asynchronous suspension>
#1      VrchatUtils.fetchAllFriends (package:zenith/util/vrchat_utils.dart:18)
<asynchronous suspension>
#2      VrchatUtils.fetchAllFriends (package:zenith/util/vrchat_utils.dart:29)
<asynchronous suspension>
#3      VrchatController._fetchFriends (package:zenith/controller/vrchat_controller.dart:121)
<asynchronous suspension>
#4      VrchatController.load (package:zenith/controller/vrchat_controller.dart:44)
<asynchronous suspension>
```

**Comment by savvazaicev** [comment]
@Rexios80 simillar issue:

`DioException [unknown]: null
Error: CheckedFromJsonException
Could not create `LimitedUser`.
There is a problem with "currentAvatarImageUrl".
Required keys are missing: currentAvatarImageUrl, currentAvatarThumbnailImageUrl.
       at FriendsApi.getFriends(friends_api.dart:339)
       at FriendsRemoteDataSource.getFriends(friends_remote_data_source.dart:12)
       at FriendsService.getFriendsPageModel(friends_service.dart:23)
       at FriendsBloc._onLoadEvent(friends_bloc.dart:29)
       at Bloc.on.<fn>.handleEvent(bloc.dart:229)`

**Comment by Rexios80** [comment]
Can you debug and get the json for that user? It's kind of weird that they don't have an avatar image.

**Comment by savvazaicev** [comment]
> Can you debug and get the json for that user? It's kind of weird that they don't have an avatar image.

Unfortunately no, i only can view it in firebase crashlytics

**Comment by Rexios80** [comment]
So it looks like pretty much every image field should be nullable?

**Comment by savvazaicev** [comment]
> So it looks like pretty much every image field should be nullable?

yes

**Comment by Rexios80** [comment]
Fixed in 1.16.3
